apps/mull-ui/src/app/pages/profile/edit-profile/edit-profile.tsx

```diff
@@ -0,0 +1,111 @@
+import { IProfileEditForm } from '@mull/types';
+import { FormikTouched, FormikValues, setNestedObjectValues, useFormik } from 'formik';
+import { isEmpty } from 'lodash';
+import React, { ChangeEvent, useState } from 'react';
+import * as Yup from 'yup';
+import { CustomFileUpload, CustomTextInput } from '../../../components';
+import { MullBackButton } from '../../../components/mull-back-button/mull-back-button';
+import MullButton from '../../../components/mull-button/mull-button';
+import './edit-profile.scss';
+
+const EditProfile = () => {
+  // TODO: Add initial profile image if user has one
+  // TODO: If user does not have an image, set to default image https://www.pngitem.com/pimgs/m/214-2145309_blank-profile-picture-circle-hd-png-download.png
```

Comment:
⚠
I don't think that's a good default profile image, something like https://t4.ftcdn.net/jpg/03/46/93/61/360_F_346936114_RaxE6OQogebgAWTalE1myseY1Hbb5qPM.jpg might be better. Please change this TODO to avoid confusion. We can discuss later what a good default profile pic would be
```suggestion
  // TODO: If user does not have an image, set to a default image
```